index.bs

```diff
@@ -0,0 +1,331 @@
+        exception.
+      </dd>
+    </dl>
+
+    If the media element's {{HTMLMediaElement/autoplayPolicy}} is different from
+    the document's {{Document/autoplayPolicy}}, the media element's
+    {{HTMLMediaElement/autoplayPolicy}} overrides the document's
+    {{Document/autoplayPolicy}}.
+
+<h2 id="code-example">Examples</h2>
+  <div class=example>
+    Example of using Document's {{Document/autoplayPolicy}}
+    <pre class="lang-javascript">
+      switch (document.autoplayPolicy) {
+        case "allowed":
+          loadUnmutedVideos();
+          break;
+        case "allowed-muted":
+          loadMutedVideos();
+          break;
+        default:
+          loadPosterImages();
+          break;
+      }
+    </pre>
+  </div>
+
+  <div class=example>
+    Example of using HTMLMediaElement's {{HTMLMediaElement/autoplayPolicy}}
+    <pre class="lang-javascript">
+      function handlePlaySucceeded() {
+        // Update the control UI to playing
+      }
+      function handlePlayFailed() {
+        // Show a button to allow users to explicitly start the video and
+        // display a image element as poster to replace the video
+      }
+
+      let video = document.getElementById("video");
+      switch (video.autoplayPolicy) {
+        case "allowed":
+          video.src = "video.webm";
+          await video.play().then(handlePlaySucceeded, handlePlayFailed);
+          break;
+        case "allowed-muted":
+          video.src = "video.webm";
+          video.muted = true;
+          await video.play().then(handlePlaySucceeded, handlePlayFailed);
+          break;
+        default:
+          // Autoplay is not allowed, no need to download the resource.
+          handlePlayFailed();
+          break;
+      }
+    </pre>
+  </div>
+
+<h2 id="security-and-privacy">Security and Privacy Considerations</h2>
+  Per the [[security-privacy-questionnaire#questions]].
+
+  The API introduced in this specification has very low impact with regards to
+  security and privacy. It does not expose any sensitive information that can be
+  used to to identify users. It does not expose any ability to control sensors
+  and any users' devices. It does not introduce any new state for an origin that
+  will persist across browsing sessions. It does not allow an origin to send any
+  data to the underlying platform. It does not introduce or enable new script
+  execution and loading mechanism. It does not allow an origin to draw over a
+  user-agent's native UI. It does not allow an origin to detect if users are in
+  the private or non-private browsing mode.
+
+<h2 id="acknowledgements">Acknowledgments</h2>
+  This specification is the collective work of the <a href="https://www.w3.org/media-wg/">W3C media Working Group</a>.
+
+  The editors would like to thank Alastor Wu, Becca Hughes, Chris Pearce,
+  Dale Curtis, Eric Carlson, Gary Katsevmanfor, Jer Noble, Mattias Buelens,
```

Comment:
Minor nit: Katsevmanfor -> Katsevman

Reply:
done, will update it later.

Reply:
And will also add you to the list, sorry for forgetting adding that in the beginning.